src/content/docs/discovery/index.mdx

```diff
@@ -0,0 +1,98 @@
+</Diagram>
+
+<Callouts>
+
+1. **Discovery**: Understand the requirements and goals of the project.
+1. **Setup**: Configure the project environment and tools.
+1. **Marketing and SEO**: Optimize the storefront for search engines and marketing campaigns.
+1. **Launch**: Deploy the storefront to a production environment.
+
+</Callouts>
+
+The discovery phase is important because Adobe Commerce is a highly customizable platform with a large thid-party extension ecosystem. Migrating an Adobe Commerce storefront to Edge Delivery Services is similar to migrating to a headless storefront (like [PWA Studio](https://developer.adobe.com/commerce/pwa-studio)).
+
+:::note[Important]
+The key requirement for a headless storefront implementation is that all required data is provided by APIs. Adobe Commerce provides all of the API coverage that you need for a successful implementation.
+:::
+
+## Existing storefronts
+
+If you have an existing Adobe Commerce storefront (Luma or PWA Studio), you can migrate it to Edge Delivery Services. The migration process involves identifying:
+
+* **Third-party extensions**: Identify the third-party extensions that you are using in your storefront. For example, do you need to connect to third-party services?
+
+* **Storefront components**: Identify the components that you want to migrate to Edge Delivery Services. For example, product detail, cart, checkout, and account pages.
+
+* **Luma Bridge options**: If you are using the Luma theme, you can use the [Luma Bridge](/discovery/luma-bridge/) to migrate your existing Luma theme to Edge Delivery Services.
+
+## Identify extensions
+
+Before starting the project, use the following list to create an inventory of the Adobe Commerce extensions that are actively being used. This will help you understand which extensions can be replaced by out-of-the-box Adobe Commerce functionality.
+
+* What extensions are currently in use?
+
+* What type of data do the extensions provide (for example, reviews)?
+
+* Is the data required on the frontend?
+    
+* How do the extensions expose the data (for example, GraphQL, REST API)?
+
+* Do the extensions expose an API to access data (for example, [Amasty Labels](https://amasty.com/docs/doku.php?id=magento_2:product_labels))? If not, create an action item to expose the required data through an API. Options include:
+
+  * Customize the Adobe Commerce Catalog Service exporter to export additional custom data to Catalog Service
+  * Create a custom Adobe Commerce GraphQL query
+  * Use [API Mesh for Adobe Developer App Builder](https://developer.adobe.com/graphql-mesh-gateway/)
+
+* Are any of the extensions for delivery options (for example, shipping/BOPIS), payments, or tax providers? If you use third-party solutions, clarify if they expose APIs on the frontend and if they provide their own set of dropin components for the frontend integration.
+
+## Validate option for dropins/widgets
+
```

Comment:
Create a plan how you will implement use cases identified for the new storefront. Adobe offers pre-built components that accelerates development: dropins and widgets